core/src/main/java/com/predic8/membrane/core/graphql/GraphQLProtectionInterceptor.java

```diff
@@ -159,6 +160,17 @@ public Outcome handleRequest(Exchange exc) throws Exception {
 
         ExecutableDocument ed = graphQLParser.parseRequest(new ByteArrayInputStream(((String) query).getBytes(UTF_8)));
 
+        int mutationCount = 0;
+        for (ExecutableDefinition definition : ed.getExecutableDefinitions()) {
+            if (definition instanceof OperationDefinition)
```

Comment:
Can you write this as stream with lambdas? There should also be a test for the count.